distribution/docs/src/main/resources/_contents/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,480 @@
+|`jks`
+|yes
+
+6+^| Global URL Properties
+
+|Protocol
+|`org.codice.${ddf-branding-lowercase}.system.protocol`
+|String
+|Default protocol that should be used to connect to this machine.
+|`https://`
+|yes
+
+|Hostname
+|`org.codice.${ddf-branding-lowercase}.system.hostname`
+|String
+|The hostname or IP address used to advertise the system. Do not enter `localhost`. Possibilities include the address of a single node or that of a load balancer in a multi-node deployment.
+
+NOTE: Does not change the address the system runs on.
+|`localhost`
+|yes
+
+|HTTPS Port
+|`org.codice.${ddf-branding-lowercase}.system.httpsPort`
+|String
+|The https port used by the system.
+
+NOTE: This *DOES* change the port the system runs on.
+|`8993`
+|yes
+
+|HTTP Port
+|`org.codice.${ddf-branding-lowercase}.system.httpPort`
+|String
+|The http port used by the system.
+
+NOTE: This *DOES* change the port the system runs on.
+|`8181`
+|yes
+
+|Default Port
+|`org.codice.${ddf-branding-lowercase}.system.port`
+|String
+|The default port used to advertise the system. This should match either the http or https port.
+
+NOTE: Does not change the port the system runs on.
+|`8993`
+|yes
+
+|Root Context
+|`org.codice.${ddf-branding-lowercase}.system.rootContext`
+|String
+|The the base or root context that services will be made available under.
+|`/services`
+|yes
+
+6+^|System Information Properties
+
+|Site Name
+|`org.codice.${ddf-branding-lowercase}.system.id`
+|String
+|The site name for ${branding}.
+|`${ddf-branding-lowercase}.distribution`
+|yes
+
+|Site Contact
+|`org.codice.${ddf-branding-lowercase}.system.siteContact`
+|String
+|The email address of the site contact.
+|
+|no
+
+|Version
+|`org.codice.${ddf-branding-lowercase}.system.version`
+|String
+|The version of ${branding} that is running.
+
+This value should not be changed from the factory default.
+|`${project.version}`
+|yes
+
+|Organization
+|`org.codice.${ddf-branding-lowercase}.system.organization`
+|String
+|The organization responsible for this installation of ${branding}.
+|Codice Foundation
+|yes
+
+6+^|Thread Pool Settings
+
+|Thread Pool Size
+|`org.codice.${ddf-branding-lowercase}.system.threadPoolSize`
+|Integer
+|Size of thread pool used for handling UI queries, federating requests, and downloading resources
+|`128`
+|Yes
+
+6+^|HTTPS Specific Settings
+
+|Cipher Suites
+|`https.cipherSuites`
+|String
+|Cipher suites to use with secure sockets
+|`TLS_DHE_RSA_WITH_AES_128_CBC_SHA,
+
+TLS_DHE_RSA_WITH_AES_128_CBC_SHA,
+
+TLS_DHE_DSS_WITH_AES_128_CBC_SHA,
+
+TLS_RSA_WITH_AES_128_CBC_SHA`
+|no
+
+|Https Protocols
+|`https.protocols`
+|String
+|Protocols to allow for secure connections
+|`TLSv1.1,TLSv1.2`
+|no
+
+|Allow Basic Auth Over Http
+|`org.codice.allowBasicAuthOverHttp`
+|Boolean
+|Set to true to allow Basic Auth credentials to be sent over HTTP unsecurely. This should only be done in a test environment. These events will be audited.
+|`false`
+|Yes
+
+|// ?
```

Comment:
This row doesn't look good... I would say we should either figure out the "// ?"'s before we merge it or delete it.